calblueprint/siren-web
# Client View Dashboard

# ✨ New in this PR ✨

## One liner: what did you do? 🚨 
Fixed up the Client View 

## Coverage 🙆‍♀️
_use this section to break up your task into submodules and track progress_ 
 - [x] Fix Intake Table so a different page's row doesn't correspond to that row of a different page
 - [x] Fix Client View so there are only tabs for cases filled out
 - [x] Fix Client Actions dropdown to correspond to unapproved/unrejected cases
 - [x] Update firebase cases when you click approve/reject
 - [x] Make Tab Bar scrollable in Client View

## How can the reviewer test your code? 👩‍💻
npm run dev

-  Fix Intake Table so a different page's row doesn't correspond to that row of a different page
     1) click to a different page of the Intake Table
     2) check that clicking on a row goes to the correct client

- Fix Client View so there are only tabs for cases filled out
     1) click through various different rows from Intake Table
     2) make sure that questions/answers appear in the tabs of their respective cases

- Fix Client Actions dropdown to correspond to unapproved/unrejected cases
     1) select a case in the dropdown of Client Actions Box that has been rejected/approved
     2) check that a message in the box states that it has been rejected/approved

- Update firebase cases when you click approve/reject
     1) select a case in the dropdown of Client Actions Box that has NOT been rejected/approved
     2) click through reject/approve and confirm
     3) check that the status in the firebase has been updated to the correct rejection/approval status

- Make Tab Bar scrollable in Client View
     1) make your viewport/window width smaller
     2) check that there are arrows in the Tab List that allow you to scroll to unseen tabs
     
<img width="608" alt="Screen Shot 2022-05-22 at 6 16 40 PM" src="https://user-images.githubusercontent.com/60284767/169725881-d6b1c166-f9f7-41eb-a3b6-7445e4c12db9.png">

## Additional Changes
- made Overview simpler:
  - tabs correspond to each case of answers ("Overview" corresponds to "general" answers)

- added new status, "resubmit":


🧜‍♀️ cc: @shannonbonet
